.travis.yml

```diff
@@ -3,13 +3,15 @@ compiler:
   - gcc
   - clang
 before_install:
-  - sudo apt-get install -y lcov libasan*
+  - sudo apt-get install -y gcc-multilib lcov libasan* libclang-3.9-dev
   - gem install coveralls-lcov
 script:
     - autoreconf -iv
     - ./configure --enable-Werror --enable-code-coverage
     - make -j8
-    - make check 2>&1 | tail -10000
-    - make distcheck 2>&1 | tail -10000
+    - make check
+    - make distcheck
```

Comment:
Without tail Travis won't save enough of the log to be useful. Try `set -o pipefail` prior to make to enable error propagation through the bash pipes.

Reply:
Yes, I already did but with tail the build terminates due to timeout

https://travis-ci.org/velichkov/asn1c/jobs/289640720#L1679-L1686

actually `make check` and `make distcheck` are redundant as `distcheck` always invoke `check` internally

Reply:
Got it, thank you.

Consider removing make check, then.